console/src/components/Services/Data/TableBrowseRows/ViewRows.js

```diff
@@ -871,6 +953,22 @@ const ViewRows = ({
     <div className={isVisible ? '' : 'hide '}>
       {getFilterQuery()}
       <div className={`row ${styles.add_mar_top}`}>
+        {selectedRows.length > 0 && (
+          <div
+            className={`${styles.display_flex}`}
+            style={{ paddingLeft: '18px' }}
+          >
+            <b className={styles.padd_small_right}>Selected:</b>
+            {selectedRows.length}
+            <button
+              className={`add_mar_right_small___3slZA btn btn-xs btn-default ${styles.bulkDeleteButton}`}
```

Comment:
You have probably copied this classname from inspect element. It won't work because these names are autogenerated on the fly while building.

You can replace `add_mar_right_small__3slZA` with `${styles.add_mar_right_small}`. It is the same classname.

Reply:
Yeah, I have no idea why it's there 🙄